src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/dashboard/ProgressVisualization.java

```diff
@@ -1,53 +1,114 @@
 package edu.kit.pse.osip.monitoring.view.dashboard;
 
+import edu.kit.pse.osip.core.model.base.AbstractTank;
+import edu.kit.pse.osip.core.utils.language.Translator;
+import java.util.Observable;
+import java.util.Observer;
+import javafx.scene.chart.LineChart;
+import javafx.scene.chart.NumberAxis;
+import javafx.scene.chart.XYChart;
+
 /**
- * Visualises a progression.
+ * Visualizes a progression.
+ * 
+ * @author Martin Armbruster
+ * @version 1.0
  */
-public class ProgressVisualization implements java.util.Observer {
+final class ProgressVisualization implements Observer {
+    /**
+     * Stores the amount of milliseconds per second.
+     */
+    private final int msPerSec = 1000;
```

Comment:
Fände es gut, wenn du das in Großbuchstaben benennen könntest. Dann wird es klarer (solche Konstanten sind immer groß, in den Projekten bei denen ich mitgearbeitet hab)